feature-libs/my-account/organization/components/unit/links/approvers/unit-approver-list.component.html

```diff
@@ -0,0 +1,146 @@
+<cx-organization-sub-list [previous]="false">
+  <a actions class="link" routerLink="../">
+    {{ 'organization.done' | cxTranslate }}
+  </a>
+</cx-organization-sub-list>
+
+<!-- <cx-view #view *ngIf="code$ | async as orgUnitCode">
```

Comment:
Can we drop unused code

Reply:
@Pucek9 Could you check if this is something old and not temp commented out?